core/utils/storage_explorer.cpp

```diff
@@ -13,43 +13,119 @@
 #include "log/configurator.hpp"
 #include "storage/trie/trie_storage.hpp"
 
+using kagome::blockchain::BlockStorage;
+using kagome::primitives::BlockHeader;
+using kagome::primitives::BlockId;
+using kagome::primitives::BlockNumber;
+using kagome::primitives::GrandpaDigest;
+using kagome::storage::trie::TrieStorage;
+
+using ArgumentList = gsl::span<const char*>;
+
+class CommandExecutionError : public std::runtime_error {
```

Comment:
Mix of error and exception confuses a bit

Reply:
Well, it's derived from runtime_error, so it's questionable. I'd rather just leave it as it is because I don't see a clear best option.

Reply:
Ok